lib/binary_to_decimal.rb

```diff
@@ -5,5 +5,44 @@
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
 def binary_to_decimal(binary_array)
+  power_num = 0
+  index = -1
+  decimal_array = []
+  decimal_number = 0
+
+  binary_array.each do |num|
+    sum_of_nums = binary_array[index] * (2 ** power_num)
+    power_num += 1
+    index -= 1
+
+    decimal_array << sum_of_nums
+  end
+
+  decimal_array.each do |num|
```

Comment:
Minor: You could use `.sum` here or have been summing each `sum_of_nums` in the loop above.